src/components/test/__snapshots__/ShortcutsModal.spec.js.snap

```diff
@@ -25,16 +25,9 @@ exports[`ShortcutsModal renders when enabled 1`] = `
           <span>
             <span
               className="keystroke"
-              key="Ctrl"
+              key="Ctrl+B"
```

Comment:
I don't know if such change is wanted? @bomsy , do you mind having a look?

Reply:
sure ... having a look

Reply:
@nchevobbe Yes, checked it out This is expected as when the tests are run the OS is unkwown, so the default changes run.